app/javascript/packs/custom_policy.js

```diff
@@ -0,0 +1,12 @@
+import {render} from 'react-dom'
+import React from 'react'
+import {CustomPolicy} from 'Policies/components/CustomPolicy'
+import {safeFromJson, parsePolicies} from 'Policies/util'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const container = document.getElementById('custom-policy-container')
+  const jsonPolicy = container.dataset.policy
+  // TODO: Notify the user something has failed. Error messages.
+  const props = (jsonPolicy) ? {policy: parsePolicies(safeFromJson(jsonPolicy, () => []))[0]} : {}
```

Comment:
second iteration we need to normalize the way we get policies from different sources and how we store them.